org.jacoco.core/src/org/jacoco/core/internal/instr/ProbeInserter.java

```diff
@@ -46,9 +47,10 @@
 	 *            callback to create the code that retrieves the reference to
 	 *            the probe array
 	 */
-	ProbeInserter(final int access, final String desc, final MethodVisitor mv,
+	ProbeInserter(final int access, final String name, final String desc, final MethodVisitor mv,
```

Comment:
For consistency we should also add JavaDoc for name parameter and document clinit member.